ext/build_defaults.yaml

```diff
@@ -21,6 +21,9 @@ build_msi:
   facter:
     ref: 'refs/tags/2.3.0'
     repo: 'git://github.com/puppetlabs/facter.git'
+  cfacter:
+    ref: 'cfacter-553c5dd5eaeda6ee6c8591ab536e776390c58f8b.zip'
+    repo: 'http://builds.puppetlabs.lan/cfacter/553c5dd5eaeda6ee6c8591ab536e776390c58f8b/artifacts/windows/cfacter-553c5dd5eaeda6ee6c8591ab536e776390c58f8b.zip'
```

Comment:
Yes, but the way we pull cfacter is different, and the others will start moving towards this. I don't want to one day have someone go "Why do we use ref and repo for our built assemblies?"